test/food.spec.js

```diff
@@ -40,4 +40,27 @@ describe('Food API', () => {
       })
     })
   })
+
+  describe('PATCH update food item path', () => {
```

Comment:
We should add tests that make sure we get a 400 back if we fail to provide either a name or the calories for the patch request. Otherwise looks good.